src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -54,9 +54,10 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
 
   renderEdition(edition) {
     const { artwork } = this.props
-    const isEcommerceEnrolled =
-      (artwork.is_acquireable || artwork.is_offerable) &&
-      (edition.is_acquireable || edition.is_offerable)
+    const artworkEcommerceAvailable =
```

Comment:
#minor - I think this does not need to be recalculated for each edition. We can do it once on artwork level and just pass it in maybe? Maybe name it something like `include_select_option` or something along those lines?

Reply:
Sounds good! :+1: